modules/statistics/RTPStatsCollector.js

```diff
@@ -573,7 +548,6 @@ StatsCollector.prototype.processStatsReport = function() {
             });
         }
 
-        // NOTE: Edge's proprietary stats via RTCIceTransport.msGetStats().
```

Comment:
I think you need to drop the code just following the comment, `msType` was the Microsoft proprietary stuff the comment mentions.

Reply:
I thought so but there was no isEdge check around it so I let it be, thanks for confirming.